runtime/moonbase/src/precompiles.rs

```diff
@@ -77,7 +78,8 @@ where
 	/// under the precompile.
 	pub fn used_addresses() -> impl Iterator<Item = R::AccountId> {
 		sp_std::vec![
-			1, 2, 3, 4, 5, 6, 7, 8, 9, 1024, 1025, 1026, 2048, 2049, 2050, 2051, 2052, 2053
+			1, 2, 3, 4, 5, 6, 7, 8, 9, 1024, 1025, 1026, 2048, 2049, 2050, 2051, 2052, 2053, 2054,
```

Comment:
why you add 254 ?

Reply:
it was missing, probably I forgot when adding the xcm-transactor

Reply:
ok